content/02.introduction.md

```diff
@@ -152,6 +152,8 @@ Other viral infections have been associated with coagulation defects; notably SA
 A wide range of viral infections can affect the coagulation cascade.
 The mechanism behind these insults has been suggested to be related to inflammation-induced increases in the von Willebrand factor (VWF) clotting protein, leading to a pro-coagulative state [@doi:10.1002/jmv.23354].
 Abnormal clotting (thromboinflammation or coagulopathy) has been increasingly discussed recently as a possible key mechanism in many cases of severe COVID-19, and may be associated with the high d-dimer levels often observed in severe cases [@doi:10/ggvd74; @doi:10.1186/s13054-020-03060-9; @doi:10.1007/s11239-020-02134-3].
+This excessive clotting in lung capillaries has been suggested to related to a dysregulated activation of the complement system, part of the innate immune system [@doi:10.1172/jci.insight.140711;@doi:10.1172/JCI140183].
```

Comment:
```suggestion
This excessive clotting in lung capillaries has been suggested to be related to dysregulated activation of the complement system, part of the innate immune system [@doi:10.1172/jci.insight.140711; @doi:10.1172/JCI140183].
```

Reply:
@yemarshall the first reference says that it's "at the interface" of innate and adaptive, the second says it's part of the innate immune system -- thought I'd ask for your thoughts!